app/views/admin/editions/_standard_fields.html.erb

```diff
@@ -3,7 +3,7 @@
   <%= render "locale_fields", form: form, edition: edition %>
   <%= form.hidden_field :lock_version %>
 
-  <% if form.object.document %>
+  <% if form.object.document&.persisted? %>
```

Comment:
Yea annoyingly though a document isn't created until an edition is persisted of the new page for some edition types.

This is a speech on the new page

<img width="616" alt="image" src="https://user-images.githubusercontent.com/42515961/192284703-b23b3202-2e76-4ada-b069-c725568b86b6.png">

Reply:
Ah, of course. That's annoying.

I guess there's always still the option of:

```suggestion
  <% if form.object.document&.slug.present? %>
```

Which would avoid the view needing to have an awareness of the `persisted?` state of the Document, and how that relates to the `slug`.

The other option to avoid an explosion would be to pass the [option `allow_nil: true` when delegating](https://apidock.com/rails/Module/delegate) `slug` on the `edition`. I think the argument could be made that it makes sense for `edition.slug` to return `nil` if it doesn't have a parent `document` yet. But that's a potentially more far reaching change.

Reply:
Yea cool i'll do that 👍